### ISSUE TITLE
make docker work with ssh private keys

### ISSUE THREAD

**Issue by pbchase** [ctsit/tt_engine#32]
Make docker work with ssh private keys

See https://medium.com/@sagargulabani/ssh-from-inside-a-dockerfile-fdfb5120d7c0
Use a volume to hold the private key file(s). 

We will mint a new key-pair for this server instance.  We will save it W/O a passphrase. Philip will do this step and send the public key to the SFTP server managers.

To take a passphrase off of a key use this command:

```
cd ~/.ssh
mkdir junk
cp id_rsa junk/private_key
cd junk
ssh-keygen -p -f private_key
# Enter the old passphrase
# Supply no passphrase for the new passphrase
```

That will do an in-place edit of the file `private_key`

**Comment by ChemiKyle** [comment]
Leaving this issue open until the feature is documented

**Comment by pbchase** [comment]
addressed by PR #33 and PR #35
